File: apply_migration.py
import os
import psycopg2

def main():
    db_url = None
    with open('apps/api/.env', 'r') as f:
        for line in f:
            if line.startswith('DATABASE_URL='):
                db_url = line.strip().split('=', 1)[1].strip('"').strip("'")
                break
                
    if not db_url:
        print("DATABASE_URL not found!")
        return
        
    conn = psycopg2.connect(db_url)
    cur = conn.cursor()
    with open('supabase/migrations/0017_fix_invitation_notifications.sql', 'r') as f:
        sql = f.read()
    cur.execute(sql)
    conn.commit()
    cur.close()
    conn.close()
    print("Success!")

if __name__ == '__main__':
    main()

File: apply_migration.js
const { Client } = require('pg');
const fs = require('fs');

async function main() {
    // Read from .env
    const env = fs.readFileSync('apps/api/.env', 'utf8');
    const dbUrlMatch = env.match(/DATABASE_URL="([^"]+)"/);
    if (!dbUrlMatch) {
        console.error("DATABASE_URL not found in apps/api/.env");
        process.exit(1);
    }
    const dbUrl = dbUrlMatch[1];
    
    const client = new Client({ connectionString: dbUrl });
    await client.connect();
    
    const sql = fs.readFileSync('supabase/migrations/0017_fix_invitation_notifications.sql', 'utf8');
    await client.query(sql);
    
    console.log("Migration applied successfully!");
    await client.end();
}

main().catch(console.error);
